code/game/objects/items/cigs_lighters.dm

```diff
@@ -863,6 +869,18 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 					e.start()
 					qdel(src)
 				return
+				if(reagents.get_reagent_amount(/datum/reagent/fuel)) // the fuel explodes, too, but much less violently

```

Comment:
Unreachable code as the line before ends the prob

Reply:
nice